requirements-test.txt

```diff
@@ -1,4 +1,6 @@
 coverage
 coveralls
+celery
 openwisp-utils[qa]>=0.2.1
 mock
+packaging
```

Comment:
same here, because it's being used in a reusable test suite which can be imported and used by other packages